README.md

```diff
@@ -200,6 +200,16 @@ Example:
 1	CACTCGTGGTACCACAAGCA
 ```
 
+## Filter cloneID
```

Comment:
My comment about "filtering" (with "ing") referred to the title of this section.

```suggestion
## Filtering cloneIDs
```